plugins/content/caption/base/lib/api/KalturaCaptionAsset.php

```diff
@@ -61,6 +61,24 @@ class KalturaCaptionAsset extends KalturaAsset
 	 */
 	public $status;
 	
+	/**
+	 * The parent id of the asset
+	 *
+	 *@var string
+	 *@readonly
+	 *
+	 */
+	public $parentId;
+	
+	/**
+	 * Is multi-language caption asset
+	 *
+	 *@var bool
+	 *@readonly
+	 *
+	 */
+	public $isMulti;	
```

Comment:
@tan-tan-kanarek @yossipapi parent_id needs to be inserted at first only , hence it's insertonly.